### ISSUE TITLE
Migrations fail when installing from pypi

### ISSUE THREAD

**Issue by nemesisdesign** [mathiasertl/django-ca#6]
It seems some additional migrations are left in the package downloaded from pypi.

I get the following error when applying migrations:

```
django.db.utils.OperationalError: table "django_ca_certificateauthority" already exists
```

**Comment by mathiasertl** [comment]
I can reproduce the issue, I'm testing a new version right now, will be released shortly.

**Comment by mathiasertl** [comment]
OK, I just released version 1.2.2, which fixes the issue (no changes otherwise).

django_ca currently has only one migration. For some reason 1.2.1 included some old migrations from before 1.0.0, they should never have been included (and were removed long ago, I don't know where setup.py picked it up.

To fix the issue on your side, please do `pip install -U django-ca`. The safest thing probably is to drop the tables in the database and delete migrations from the migrations table and just do the migrations again. Since you just installed, I assume you don't have any data.

If you need further help, please let me know!

**Comment by mathiasertl** [comment]
Seems to work now? ^^
